doc/mgr/prometheus.rst

```diff
@@ -184,6 +185,18 @@ Example to turn up the sync interval to 10 minutes::
 
   ceph config set mgr mgr/prometheus/rbd_stats_pools_refresh_interval 600
 
+
+Ceph daemons performance counters metrics
+-----------------------------------------
+
+With the introduction of ceph-exporter daemon, the module will no longer export ceph daemons
+perf counters as prometheus metrics by default. Although with the module option ``fetch_perf_counters_metrics`` you can still
+enable support for exporting these metrics. By default, the value of config will be ``false``.
+To enable this module option, simply set config key to ``true``::
```

Comment:
@anthonyeleven Nothing changes!  It's just the new source for exporting perf counters as metrics instead of prometheus module, as it's not performant at scale; with large clusters (>4000 OSDs). 
All this detail will be summarized in the documentation of ceph-exporter, which is still a WIP. 
Do you think that it's documentation should be delivered before the changes made in prometheus doc here? In that case maybe I can make a separate PR including documentation for both ceph-exporter and prometheus. Wdyt?

Reply:
Yes, there will be  two main sources of metrics:
- Prometheus manager module: It is responsible for exposing all metrics other than ceph daemons performance counters. (port `9283`)
- Ceph exporter: It is responsible for exposing only ceph daemons performance counters as prometheus metrics. (`9926`)

Reply:
I'm having difficulty reconciling that with "Nothing changes! ".  I request a sentence like "These metrics are now available from the new `ceph_exporter`endpoint on 9926/TCP in the same Prometheus exposition format.  Details can be found in the `ceph_exporter` documentation [here]

Reply:
> I'm having difficulty reconciling that with "Nothing changes! ". I request a sentence like "These metrics are now available from the new `ceph_exporter`endpoint on 9926/TCP in the same Prometheus exposition format. Details can be found in the `ceph_exporter` documentation [here]

I'm gonna remove the doc changes from this PR, not to block the current changes which are supposed to go in and not related to the changes in prometheus module. I'll address the module changes and the updated documentation for module and ceph-exporter in a separate PR.

Reply:
I've created 2 trackers : 1 for new exporter documentation and other for changes required in prometheus module and it's documentation.
https://tracker.ceph.com/issues/58163
https://tracker.ceph.com/issues/58164